src/oc/email/invite.clj

```diff
@@ -0,0 +1,12 @@
+(ns oc.email.invite
+  (:require [clojure.string :as s]
+            [hiccup.core :as h]))
+
+(defn text [msg]
```

Comment:
Maybe for phase 2 but we should change this to be html and add a tracking pixel to report "email opened" in the invitations. i remember it was very helpful when inviting people to testflight to see if they lost the email or they simply didn't care about it..